subsys/ipc/ipc_service/backends/ipc_icbmsg.c

```diff
@@ -1117,10 +1121,15 @@ static int backend_init(const struct device *instance)
 {
 	const struct icbmsg_config *conf = instance->config;
 	struct backend_data *dev_data = instance->data;
+	static K_THREAD_STACK_DEFINE(ep_bound_work_q_stack, EP_BOUND_WORK_Q_STACK_SIZE);
 
 	dev_data->conf = conf;
 	dev_data->is_initiator = (conf->rx.blocks_ptr < conf->tx.blocks_ptr);
 	k_mutex_init(&dev_data->mutex);
+	k_work_queue_init(&dev_data->ep_bound_work_q);
+	k_work_queue_start(&dev_data->ep_bound_work_q, ep_bound_work_q_stack,
+			   K_THREAD_STACK_SIZEOF(ep_bound_work_q_stack),
+			   CONFIG_SYSTEM_WORKQUEUE_PRIORITY, NULL);
```

Comment:
Can we add a `#define` for the priority to avoid directly referring to system workq priority here?